Resources/views/Collector/icon.svg

```diff
@@ -0,0 +1,7 @@
+<svg viewBox="9.954 10.02 33.268 32.669" width="33.268" height="32.669">
+    <polygon fill="#AAA" points="10.204000473022461 33.944000244140625 20.1299991607666 33.891998291015625 21.15399932861328 37.93600082397461 22.152000427246094 33.917999267578125 27.457000732421875 33.970001220703125 26.195999145507812 39.43299865722656 32.6150016784668 39.43299865722656 37.97100067138672 39.43299865722656 36.59400177001953 33.99700164794922 43.00199890136719 33.891998291015625 43.00199890136719 42.63600158691406 10.125 42.68899917602539" />
+    <polygon fill="#AAA" points="9.954000473022461 10.020000457763672 42.803001403808594 10.020000457763672 42.803001403808594 18.70800018310547 32.88399887084961 18.684999465942383 31.909000396728516 14.734999656677246 31.000999450683594 18.66200065612793 25.56599998474121 18.70800018310547 26.889999389648438 13.24899959564209 21.34000015258789 13.24899959564209 15.060999870300293 13.24899959564209 16.43600082397461 18.70800018310547 10.00100040435791 18.684999465942383" />
```

Comment:
Creating the icon aligned on a 24px grid as the core profiler icons could help avoiding such weird coordinates.